.gitignore

```diff
@@ -57,6 +57,9 @@ ide/nbproject/private
 nb-configuration.xml
 nbactions.xml
 
+#VS code
```

Comment:
i would like to create a new jira on change the gitingore , as this code has nothing to do with the pr.
But I'd also like to hear other people's opinions here.